lib/style/_tech-mixins.scss

```diff
@@ -107,6 +107,9 @@
 }
 
 @mixin ellipsis {
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
Could you use here @internet-explorer mixin also?

Reply:
I didn't want to access here mixin defined in src directory. Can I move `lib/style/_tech-mixins.scss` to `src/client/styles/_tech-mixins.scss` ?

Reply:
Good point. Sure, move it.